src/components/Layout/LayoutStyles.js

```diff
@@ -147,6 +163,10 @@ const fixedSidebarButton = css`
     border: 3px solid ${pink};
     margin: -9px -3px;
   }
+
+  @media (min-width: 820px) {
+    display: none;
```

Comment:
Was this missed originally? 🤔

Surprised I didn't notice the sidebar button being visible on larger screens if it was; if not then I guess something I changed here caused that... but no idea what that would be! 😂